Dockerfile

```diff
@@ -7,15 +7,16 @@ USER       root
 RUN        yum install -y epel-release
 RUN        yum repolist
 RUN        yum install -y git python-pip python-devel
+RUN	   yum install -y gcc openssl-devel
 RUN        pip install --upgrade pip
 RUN        useradd -d /home/flasker -m flasker
 RUN        mkdir /dist
 
 # Must run python setup.py sdist first.
 ARG        VERSION="0.0.1"
 LABEL      version="$VERSION"
-COPY       dist/sqre-uservice-ltdstatus-$VERSION.tar.gz /dist
-RUN        pip install /dist/sqre-uservice-ltdstatus-$VERSION.tar.gz
+COPY       dist/sqre-uservice-ccutter-$VERSION.tar.gz /dist
+RUN        pip install /dist/sqre-uservice-ccutter-$VERSION.tar.gz
```

Comment:
Initially VERSION is set from cookiecutter.  It might be nice to have setup/Dockerfile/the version in server.py all follow the same thing, but I haven't worked through a nice way to get all three of them.  You can do server.py tracking setup with setuptools, but propagating to Dockerfile...well, Dockerfile can't run-a-thing-and-include-its-output, can it?

Future enhancement.  It starts at the same place.  I can add some notes to the cookiecutter docs that says "as you evolve this, you need to change the app version in three places: x, y, z"